paint-timing/fcp-only/fcp-bg-image-two-steps.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<head>
+<title>Performance Paint Timing Test: FCP due to background image, in two steps</title>
+<style>
+    #main {
+        width: 100px;
+        height: 100px;
+    }
+
+    .intermediate {
```

Comment:
Intermediate is not a very useful name because it does not capture the fact that FCP should still not be present at this stage. Consider renaming... something like `stillNoFCP` (I'm not good with naming either)